pipelines/prep_data.py

```diff
@@ -217,26 +217,24 @@ def process_state_level_data(
 
     disease_key = _disease_map.get(disease, disease)
 
-    if state_abb == "US":
+    if loc_abb == "US":
         locations_to_aggregate = (
-            state_pop_df.filter(pl.col("abb") != "US")
-            .get_column("abb")
-            .unique()
+            loc_pop_df.filter(pl.col("abb") != "US").get_column("abb").unique()
         )
         logger.info("Aggregating state-level data to national")
-        state_level_nssp_data = aggregate_to_national(
-            state_level_nssp_data,
+        loc_level_nssp_data = aggregate_to_national(
+            loc_level_nssp_data,
             locations_to_aggregate,
             first_training_date,
             national_geo_value="US",
         )
 
     return (
-        state_level_nssp_data.filter(
+        loc_level_nssp_data.filter(
             pl.col("disease").is_in([disease_key, "Total"]),
             pl.col("metric") == "count_ed_visits",
-            pl.col("geo_value") == state_abb,
-            pl.col("geo_type") == "state",
+            pl.col("geo_value") == loc_abb,
+            pl.col("geo_type") == "loc",
```

Comment:
This is "state" in the actual table, no?
```suggestion
            pl.col("geo_type") == "state",
```